<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng Nhập Hệ Thống</string>
  </property>
  <property name="styleSheet">
   <string>background-color: #e0f7fa;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- Nhãn tiêu đề -->
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Đăng Nhập Hệ Thống</string>
    </property>
    <property name="styleSheet">
     <string>font-size: 18px; font-weight: bold; color: #00796b;</string>
    </property>
   </widget>
   <!-- Nhãn và trường nhập liệu -->
   <widget class="QLabel" name="lblTenDangNhap">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên đăng nhập:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTenDangNhap">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nhập tên đăng nhập</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMatKhau">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mật khẩu:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMatKhau">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Nhập mật khẩu</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
   </widget>
   <!-- Nút đăng nhập -->
   <widget class="QPushButton" name="btnDangNhap">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Đăng nhập</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #00796b; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
   </widget>
   <!-- Nhãn trạng thái -->
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="styleSheet">
     <string>color: #d32f2f;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>